src/panels/config/integrations/dialog-add-integration.ts

```diff
@@ -509,7 +509,14 @@ class AddIntegrationDialog extends LitElement {
     if (integration.integrations) {
       const integrations =
         this._integrations![integration.domain].integrations!;
-      this._fetchFlowsInProgress(Object.keys(integrations));
+      let domains = Object.keys(integrations);
+      if (integration.iot_standards?.includes("homekit")) {
+        domains.push("homekit_controller");
+      }
+      if (integration.domain === "apple") {
```

Comment:
On mobile so can't see the full code, but we still want to allow people to start a normal homekit controller flow from apple screen. Just hide the discovered ones

Reply:
yeah, thats what this does, it filters the discovery